libs/deps/merch-offers.js

```diff
@@ -0,0 +1,95 @@
+// Mon, 04 Dec 2023 20:07:05 GMT
+import{html as d,LitElement as h}from"./lit-all.min.js";import{LitElement as c}from"./lit-all.min.js";var s=class extends c{static properties={offers:{type:Array},selectedOffer:{type:Object},defaults:{type:Object}};saveContainerDefaultValues(){let e=this.closest(this.getAttribute("container")),t=e?.querySelector('[slot="description"]:not(merch-offer > *)')?.cloneNode(!0),i=e?.badgeText;return{description:t,badgeText:i}}getSlottedElement(e,t){return(t||this.closest(this.getAttribute("container"))).querySelector(`[slot="${e}"]:not(merch-offer > *)`)}updateSlot(e,t){let i=this.getSlottedElement(e,t);if(!i)return;let o=this.selectedOffer.get(e)?this.selectedOffer.get(e):this.defaults[e];o&&i.replaceWith(o.cloneNode(!0))}updateBadgeText(e){this.selectedOffer.badgeText===""?e.badgeText=null:this.selectedOffer.badgeText?e.badgeText=this.selectedOffer.badgeText:e.badgeText=this.defaults.badgeText}updateContainer(){let e=this.closest(this.getAttribute("container"));!e||!this.selectedOffer||(this.updateSlot("cta",e),this.updateSlot("price",e),this.updateSlot("description",e),this.updateBadgeText(e))}selectOffer(e){let t=this.selectedOffer,i=e.detail;i&&(t&&(t.selected=!1),this.selectedOffer=i,this.updateContainer())}connectedCallback(){this.offers=[...this.querySelectorAll("merch-offer")],this.defaults=this.saveContainerDefaultValues(),this.addEventListener("offer-selected",this.selectOffer),this.offers[0].select(),this.selectedOffer=this.offers[0]}disconnectedCallback(){this.removeEventListener("offer-selected",this.selectOffer)}};customElements.define("merch-offers",s);import{css as l}from"./lit-all.min.js";var n=l`
+    :host {
```

Comment:
seems like file rename didn't work properly, this file can be deleted, its not used

Reply:
ah, i know why it could have happened, you probably moved all /libs/* build result from taco to milo deps?
could be that build in taco didn't remove stale files

Reply:
I need to remove the one